db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended to check this file into your version control system.
 
-ActiveRecord::Schema.define(:version => 20150904055454) do
+ActiveRecord::Schema.define(:version => 20151015024144) do
```

Comment:
Dealing with schemas in git is always a pain. If you're free, I can take you through how you might clean this up.